src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -3260,23 +3267,22 @@ private static string AppendFloatSuffix(string memberValue)
                     .Aggregate((s1, s2) => "{0},{1}".InvariantCultureFormat(s1, s2));
         }
 
-        private string BuildColor(string memberValue)
-        {
-            var colorHelper = (INamedTypeSymbol)_medataHelper.GetTypeByFullName(XamlConstants.Types.Colors);
-
-            if (colorHelper.GetFields().Any(m => m.Name.Equals(memberValue, StringComparison.OrdinalIgnoreCase)))
-            {
-                return $"{XamlConstants.Types.Colors}.{memberValue}";
-            }
-            else
-            {
-                memberValue = string.Join(", ", ColorCodeParser.ParseColorCode(memberValue));
+		private string BuildColor(string memberValue)
+		{
+			var colorHelper = (INamedTypeSymbol)_medataHelper.GetTypeByFullName(XamlConstants.Types.Colors);
+			if (colorHelper.FindField(XamlConstants.Types.Color, memberValue, StringComparison.OrdinalIgnoreCase) is IFieldSymbol field)
```

Comment:
Perform an `INamedTypeSymbol` at the construction of the generator for `XamlConstants.Types.Color`.

Reply:
Should we also cache `XamlConstants.Types.Colors` at ctor as well?
Or, use the memoized version with `GetType`?